packages/lib-classifier/src/components/Classifier/components/ImageToolbar/ImageToolbar.js

```diff
@@ -20,13 +20,13 @@ class ImageToolbar extends Component {
     return (
       <aside {...props}>
         <Box
-          background={ mode === 'light' ? 'white' : '#333' }
+          background={mode === 'light' ? 'white' : '#333'}
```

Comment:
Should this be a named colour in the theme?

Reply:
Oh wait, it is: `darkestGrey`

Reply:
My names aren't fantastic, heh. When Becky gets back, I'm going to ask her to add the colors to https://projects.invisionapp.com/dsm/zooniverse/primary-brand/folder/colors/5bbd0dbcd018e900118186e8 for easier reference with the mapped grommet name.